.github/workflows/release_check.yml

```diff
@@ -0,0 +1,29 @@
+name: Run JS SDK test
```

Comment:
The workflow name 'Run JS SDK test' seems incorrect as this workflow is actually checking Python type stubs. Consider renaming it to something more appropriate like 'Check Python Type Stubs' or 'Verify Type Definitions'.